Lib/test/test_free_threading/test_type.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+
+from threading import Thread
+from multiprocessing.dummy import Pool
+from unittest import TestCase
+
+from test.support import is_wasi
+
+
+NTHREADS = 6
+BOTTOM = 0
+TOP = 1000
+ITERS = 100
```

Comment:
How long does this test take to run in a debug build? The `test_free_threading` tests are a bit slow right now (~1 min in debug builds on Linux, I think). I think we should aim for the default to be fast, even if that means it's less likely to catch bugs. (We can add flag or option to run for longer).

Reply:
Currently 9 seconds total (`test_attr_cache` is actually pretty fast, the other two are slower).  I'll reduce the loop length in the other 2 and which gets it down to 2-3 seconds.